readme.md

```diff
@@ -94,7 +114,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+The concept is similar in that there is an exchange system that makes sure that you are the owner of your car before it is brought to you. You pull up and hand over your key and you get a ticket(token). Your car goes somewhere and you hope they dont adjust you settings. To get the car back you hand them your ticket(token) and they check it against their ticket list to see if it matches. If it does then a request to get your car is made and it is delivered to you.
```

Comment:
:+1:

Plus, each has limited access (valet key doesn't work for trunk/glovebox).